.github/workflows/django.yml

```diff
@@ -0,0 +1,30 @@
+name: Django CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
please change this to allow the workflow to be executed on push to any branches, this workflow does ui testing and code style checks so it should not be bounded to an specific branch.

`on: [push]`